src/System.Private.Xml/tests/XmlDocument/XmlAttributeCollectionTests/InsertAfterTests.cs

```diff
@@ -230,6 +231,7 @@ public void InsertAfterRemovesDupRefAttrAtTheEnd()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] //Fix for this edge case was only made in NetCore
```

Comment:
Test and behavioral change introduced by https://github.com/dotnet/corefx/issues/1305.